storage/remote/client.go

```diff
@@ -37,18 +37,16 @@ const maxErrMsgLen = 256
 
 // Client allows reading and writing from/to a remote HTTP endpoint.
 type Client struct {
-	index      int // Used to differentiate metrics.
-	url        *config.URL
-	client     *http.Client
-	timeout    time.Duration
-	readRecent bool
+	index   int // Used to differentiate metrics.
```

Comment:
Maybe unrelated but small enough fix to include: the index differentiates remote read clients, not metrics.

Reply:
Yeah my comments not clear - its differentiates remote [read/write] clients in the metrics Prometheus exports.